File: Selenium/Activities/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:

    driver.get("https://v1.training-support.net/selenium/target-practice")

    print("Title of the page: ",driver.title)

    third_header = driver.find_element(By.XPATH,"//h3[@id='third-header']")

    print("Text of the 3rd header: ",third_header.text)

    fifth_header = driver.find_element(By.XPATH,"//h5[text()='Fifth header']")

    print("Color of the 5th header: ",fifth_header.value_of_css_property("color"))

    violet_button = driver.find_element(By.XPATH,"//button[text()='Violet']")

    print("Classes of the Violet button: ",violet_button.get_attribute("class"))
    
    grey_button = driver.find_element(By.CLASS_NAME,"grey")
		
    print("Text of the Grey button: " + grey_button.text)

    driver.quit()

File: Selenium/Activities/Activity4.java
package com.example.fst_m1_selenium;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity4 {
	
	public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		
		driver.get("https://v1.training-support.net/selenium/target-practice");
		
		System.out.println("Title of the page: " + driver.getTitle());
		
		String headertxt= driver.findElement(By.xpath("//h3[@id='third-header']")).getText();
		
		System.out.println("Text of the 3rd header: " + headertxt);
		
		String color = driver.findElement(By.xpath("//h5[text()='Fifth header']")).getCssValue("color");
		
		System.out.println("Color of the 5th header: " + color);
		
		String violetButtonClass = driver.findElement(By.xpath("//button[text()='Violet']")).getAttribute("class");
		
		System.out.println("Classes of the Violet button: " + violetButtonClass);
		
		String text = driver.findElement(By.className("grey")).getText();
		
		System.out.println("Text of the Grey button: " + text);
		
		driver.quit();
	}

}
